grp/simplerew.grp

```diff
@@ -20964,6 +20964,1175 @@ return
 [19699460169550,
 3271674397651512456087713125906584073460939319520448043695952320284114\
 735552723852527381679661909541468882928650336526651532801441553]]],
+[[3265920,"2A",[3,3],"U4(3)"],
+[(1,5,8,10)(2,6,7,11)(3,4,9,12)(13,26,16,19)
```

Comment:
I am not so happy about this list of data files without any information about how to reproduce the data, where it comes from, and how to verify that it is correct. Not that I mistrust you as a source, to the contrary, I have full confidence -- but it feels a bit like writing in a paper "Proof: this lemma is true because my colleague AH once told me he proved it and I trust him". Might be good enough for me, but maybe not the referee...

Anyway, I am not calling for any action here right now, let's merge this -- but it would be good if we could eventually come up with a plan to change this situation!